_events/sr2019/competition.md

```diff
@@ -12,9 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2019, showcasing
 
 The competition will be held from *Saturday 6th April* until *Sunday 7th April*. Doors open 9am and we aim to finish around 6pm.
 
-The Saturday will begin with a briefing on the rules and general struture of the weekend, followed by some time for teams to test their robots in the arena. Soon after, the league matches will start, in which your robot will compete against 3 other bots to rise up the league. These continue for the rest of the day.
-
-On Sunday morning, the league matches will continue. Once these have concluded, we will begin knockout matches, the order of which is determined by your position in the league. The remaining teams will compete in the finals, followed by the awards for the winning teams.
```

Comment:
Any reason not to leave this in place as well as the tables? The prose is somewhat nicer to read (and I suspect somewhat friendlier to both news organisations and screen readers).

Reply:
(perhaps pull the paragraphs under each of the new headings though)